app/src/main/res/values-night/themes.xml

```diff
@@ -0,0 +1,16 @@
+<resources xmlns:tools="http://schemas.android.com/tools">
```

Comment:
Do we need to support the `dark themes`?

Reply:
Auto-generated by AS Chipmunk.